projects/core/components/hosted-dropdown/hosted-dropdown.component.ts

```diff
@@ -184,6 +197,11 @@ export class TuiHostedDropdownComponent implements TuiFocusableElementAccessor {
         }
     }
 
+    @HostListener('mouseover', ['$event.target'])
```

Comment:
This will spam change detection as mouse moves across nested tags. How about we use `tuiTypedFromEvent(this.el.nativeElement, 'mouseover')` instead of `onHoverTarget$`?